h1b/src/usb/mod.rs

```diff
@@ -23,17 +23,30 @@ pub mod u2f;
 
 use cortexm3::support;
 use kernel::ReturnCode;
+use kernel::common::registers::LocalRegisterCopy;
 
 pub use self::constants::Descriptor;
+pub use self::registers::AhbConfig;
+pub use self::registers::AllEndpointInterrupt;
+pub use self::registers::DeviceConfig;
+pub use self::registers::DeviceControl;
 pub use self::registers::DMADescriptor;
+pub use self::registers::EndpointControl;
+pub use self::registers::Gpio;
+pub use self::registers::InEndpointInterruptMask;
+pub use self::registers::Interrupt;
+pub use self::registers::OutEndpointInterruptMask;
+pub use self::registers::Reset;
+pub use self::registers::UsbConfiguration;
```

Comment:
It may be worth doing `pub use self::registers::{AhbConfig, AllEndpointInterrupt, Descriptor, ...};` here.

Reply:
Good point -- I've reformatted all of the use statements to be cleaner.